pkg/server/auth/azure.go

```
@@ -41,9 +41,9 @@ func GetAzureOAuthConfig() *oauth2.Config {
 	}
 }
 
-// GetAzureUser fetches user info from Microsoft Graph API using the access token
-func GetAzureUser(token *oauth2.Token) (*AzureUser, error) {
-	client := GetAzureOAuthConfig().Client(context.Background(), token)
+// GetAzureUser fetches user info from Microsoft Graph API using the access token.
+// client 由调用方传入（携带 OAuth 代理与超时配置）。
+func GetAzureUser(client *http.Client) (*AzureUser, error) {
 	resp, err := client.Get("https://graph.microsoft.com/v1.0/me")
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)